docs/editor_guide/creating_personalized_content.rst

```diff
@@ -0,0 +1,91 @@
+Creating personalized content
+=============================
+
+Once you've created a segment you can start serving personalised content to your
+visitors. To do this, you can go one of three directions.
+
+1. Create a variant of a page for your segment.
+2. Use StreamField blocks only visible to your segment.
+3. Use a template block only visible to your segment.
```

Comment:
kan denk ik meer versimpeld worden:

 +1. Create a page variant for a segment.
 +2. Use StreamField blocks visible for a segment only.
 +3. Use a template block visible for a segment only.